app/src/main/java/com/mealsmadeeasy/data/FakeMealStore.kt

```diff
@@ -164,4 +166,41 @@ class FakeMealStore : MealStore {
             else -> return Single.just(iceCream)
         }
     }
+
+    override fun getRecipe(id: String): Single<Recipe> {
+        val tacosRecipe = Recipe(
+                prepTime = 20,
+                steps = listOf(
+                        RecipeStep("Cut chicken into 1/2 inch strips."),
+                        RecipeStep("Heat oil in large skillet over medium-high heat, adding chicken once hot. " +
+                                "Cook and stir for 10 minutes or until cooked through."),
+                        RecipeStep("Spoon 1/3 of the chicken into each tortilla and sprinkle with cheese. " +
+                                "Top evenly with lettuce and tomato.")
+                ),
+                ingredients = listOf(
+                        Ingredient(id = "taco-shell", name = "Flour tortilla shells", quantity = 3, unitName = ""),
+                        Ingredient(id = "taco-meat_chicken", name = "Chicken", quantity = 0.5, unitName = "lbs"),
+                        Ingredient(id = "shredded_cheese", name = "Shredded cheese", quantity = 0.75, unitName = "Cups"),
+                        Ingredient(id = "lettuce", name = "Lettuce", quantity = 2, unitName = "Leaves"),
+                        Ingredient(id = "tomato", name = "", quantity = 0.5, unitName = "Tomatoes")
+                )
+        )
+
+        val iceCreamRecipe = Recipe(
+                prepTime = 5,
+                steps = listOf(
+                        RecipeStep("Take the ice cream out of the freezer, letting it thaw for 5 minutes."),
```

Comment:
People let ice cream thaw? Who has time for this?

(This can totally stay, I'm just slightly amused with your ice cream recipe)